realm/realm-library/src/objectServer/java/io/realm/internal/objectserver/SyncSession.java

```diff
@@ -87,7 +86,6 @@
  *
  * This object is thread safe.
  */
-@Keep
```

Comment:
If we want to call `notifySessionError` we need to keep this class unobfuscated.